setup.cfg

```diff
@@ -31,7 +31,7 @@ install_requires =
     fastapi>=0.68.0
     pydantic>=1.8.0
     uvicorn>=0.15.0
-    git+https://github.com/pachterlab/ffq@devel
+    ffq @ git+https://github.com/pachterlab/ffq@1d77aeaa
```

Comment:
I guess this is the `setup.cfg` equivalent of doing `#egg=ffq` in `setup.py`?

Reply:
I guess so. I didn't even realize it was different from running `pip install` directly or using a requirements file. I just assumed pip would infer everything from the git repo.

Reply:
Python packaging is a bit of a mess for this stuff 😞

Reply:
Should I merge it? 

<img src="https://media2.giphy.com/media/hv53DaYcXWe3nRbR1A/giphy.gif"/>

Reply:
yes. I tested more thoroughly this time.

Reply:
vamos!